src/Aspire.ProjectTemplates/templates/aspire-apphost/Properties/launchSettings.json

```diff
@@ -1,5 +1,5 @@
 {
-  "$schema": "http://json.schemastore.org/launchsettings.json",
+  "$schema": "https://json.schemastore.org/launchsettings.json",
```

Comment:
Have someone validated that these changes work as expected with a project created from these templates? @eerhardt @DamianEdwards ?

Reply:
Yes. I have Aspire.ProjectTemplates `8.0.0-preview.5.24127.8` installed locally (which has these changes from main). And it works just fine.

Note that these launchSettings.json changes are already in the aspnetcore repo. For example: https://github.com/dotnet/aspnetcore/pull/52203